File: samples/client/petstore/jaxrs-cxf-client/src/main/java/io/swagger/api/impl/StoreApiServiceImpl.java
package io.swagger.api.impl;

import io.swagger.api.*;
import java.util.Map;
import io.swagger.model.Order;

import java.io.InputStream;
import java.io.OutputStream;
import java.util.List;
import java.util.Map;
import javax.ws.rs.*;
import javax.ws.rs.core.Response;
import org.apache.cxf.jaxrs.model.wadl.Description;
import org.apache.cxf.jaxrs.model.wadl.DocTarget;

import org.apache.cxf.jaxrs.ext.multipart.*;

import io.swagger.annotations.Api;

public class StoreApiServiceImpl implements StoreApi {
    public void deleteOrder(String orderId) {
        // TODO: Implement...
        
        
    }
    
    public Map<String, Integer> getInventory() {
        // TODO: Implement...
        
        return null;
    }
    
    public Order getOrderById(Long orderId) {
        // TODO: Implement...
        
        return null;
    }
    
    public Order placeOrder(Order body) {
        // TODO: Implement...
        
        return null;
    }
    
}

File: samples/server/openapi3/petstore/kotlin-springboot-reactive/src/main/kotlin/org/openapitools/api/StoreApiServiceImpl.kt
package org.openapitools.api

import org.openapitools.model.Order
import kotlinx.coroutines.flow.Flow;
import org.springframework.stereotype.Service
@Service
class StoreApiServiceImpl : StoreApiService {

    override suspend fun deleteOrder(orderId: String): Unit {
        TODO("Implement me")
    }

    override suspend fun getInventory(): Map<String, Int> {
        TODO("Implement me")
    }

    override suspend fun getOrderById(orderId: Long): Order {
        TODO("Implement me")
    }

    override suspend fun placeOrder(order: Order): Order {
        TODO("Implement me")
    }
}
